### DISCUSSION TITLE
Warning of Epoch

### DISCUSSION THREAD

**Discussion by yinliru** [epochpic/epoch#327] [Q&A]
Dear friends, 
I have a question. There is a "warning" in my "**nohup.out**" file, as  shown "Output string "**Output string EPOCH input deck: /home/ylr/epoch4179/epoch2d/Data/ylr/Density gradient/up/1/input.deck**" has been truncated", but I don’t know why it is truncated. Does anyone know what causes this problem? Looking forward to your answers, thank you.

Sincerely
![微信截图_20211226231507](https://user-images.githubusercontent.com/96190882/147412629-0d400750-836e-49c8-a90a-9e70574058d2.png)

**Comment by liuxybeta** [comment]
Well, I have the same question. I want to know whether this warning message needs to be processed, and how to eliminate this warning. Thanks for your help. @TomGoffrey
